docs/preset-env.md

```diff
@@ -229,38 +229,64 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-> This option adds direct references to the `core-js` module as bare imports. Thus `core-js` will be resolved relative to the file itself and needs to be accessible. You may need to specify `core-js@2` as a top level dependency in your application if there isn't a `core-js` dependency or there are multiple versions.
+> When using this options, you also need to configure the [`corejs`](#corejs) option.
 
 This option configures how `@babel/preset-env` handles polyfills.
 
+This option adds direct references to the `core-js` module as bare imports. For this reason, you need to install `core-js@3` or `core-js@2` (depending on the value of the [`corejs`](#corejs) option) as a dependency of your application:
+
+```sh
+npm install core-js@3 --save
+```
+
 #### `useBuiltIns: 'entry'`
 
-> NOTE: Only use `require("@babel/polyfill");` once in your whole app.
-> Multiple imports or requires of `@babel/polyfill` will throw an error since it can cause global collisions and other issues that are hard to trace.
-> We recommend creating a single entry file that only contains the `require` statement.
+> NOTE: Only use `import "core-js";` and `import "regenerator-runtime/runtime";` once in your whole app.
```

Comment:
I think we need to rework this section to note that while `@babel/polyfill` is deprecated, it still works?